controllers/host/platformnetworks.go

```diff
@@ -439,11 +437,34 @@ func (r *HostReconciler) IsReconfiguration(client *gophercloud.ServiceClient, ne
 	return nil, false
 }
 
-// ShouldReconcile is a very important function that really controls the reconciliation
+func (r *HostReconciler) ReconcileStaticPlatformNetworkExpected(client *gophercloud.ServiceClient,
+	network_instance *starlingxv1.PlatformNetwork,
+	addrpool_instance *starlingxv1.AddressPool,
+	uuid string) (error, bool) {
+
+	if addrpool_instance != nil {
+		err, is_reconfig := r.IsReconfiguration(client, network_instance, addrpool_instance)
+		if err != nil {
+			return err, false
+		}
+		if !is_reconfig {
+			return nil, true
+		}
+	} else {
+		// for networks
+		if uuid == "" {
+			return nil, true
+		}
+	}
+
+	return nil, false
+}
+
+// ReconcilePlatformNetworkExpected is a very important function that really controls the reconciliation
 // behaviour of network and associated addresspools. Note that parameters 'update_required'
 // and 'uuid' refers to address pool update_required and address pool uuid when called from
 // ReconcileAddrPoolResource function.
-func (r *HostReconciler) ShouldReconcile(client *gophercloud.ServiceClient, network_instance *starlingxv1.PlatformNetwork, addrpool_instance *starlingxv1.AddressPool, update_required bool, uuid string) (error, bool) {
+func (r *HostReconciler) ReconcilePlatformNetworkExpected(client *gophercloud.ServiceClient, network_instance *starlingxv1.PlatformNetwork, addrpool_instance *starlingxv1.AddressPool, update_required bool, uuid string) (error, bool) {
```

Comment:
rename the uuid as "addrpool_uuid"

Reply:
Actually, the method is commonly used for both Network as well as AddressPool reconciliation so the UUID applies to both.

Reply:
tbh, it is quite confusing for other devs, can you update the function description? I can only find the address pool uuid in the description. Also can you update the "uuid" variable key name in the place calling this method with network_uuid and addrpool_uuid respectively?